### PR TITLE
Issue58

### PR DESCRIPTION
**Pull request by yakobi99**
Added gazebo plugins for d435i and GPS. Modified protobot_rviz.launch

### CODE REVIEW

**Comment by yakobi99** [comment]
just merge the urdf.xacro file

**Comment by yakobi99** [comment]
ok merge conflicts have been resolved

File: src/protobot_description/urdf/proto_protobot.urdf.xacro [hunk 1]
```diff
@@ -73,4 +73,56 @@
...
+                <Cx>0</Cx>
+                <Cy>0</Cy>
+                <focalLength>0</focalLength>
+                <hackBaseline>0</hackBaseline>
+            </plugin>
+        </sensor>
+    </gazebo>
+
+    <!-- Implement t265_link -->
+
```
Comment (bretwitt) [inline src/protobot_description/urdf/proto_protobot.urdf.xacro:127]: Good opportunity to integrate this plugin https://github.com/pal-robotics/realsense_gazebo_plugin

**Review by bretwitt — COMMENTED** [review]
See comment

https://github.com/pal-robotics/realsense_gazebo_plugin

**Pull request closed without merging**